t/plugin/grpc-web.t

```diff
@@ -68,25 +68,33 @@ passed
 
 
 
-=== TEST 2: Proxy unary request using APISIX gRPC-Web plugin
+=== TEST 2: Proxy unary request using APISIX with trailers gRPC-Web plugin
 --- exec
 node ./t/plugin/grpc-web/client.js BIN UNARY
 node ./t/plugin/grpc-web/client.js TEXT UNARY
 --- response_body
+Status: { code: 0, details: '', metadata: {} }
+Status: { code: 0, details: '', metadata: {} }
 {"name":"hello","path":"/hello"}
+Status: { code: 0, details: '', metadata: {} }
+Status: { code: 0, details: '', metadata: {} }
 {"name":"hello","path":"/hello"}
 
 
 
-=== TEST 3: Proxy server-side streaming request using APISIX gRPC-Web plugin
+=== TEST 3: Proxy server-side streaming request using APISIX with trailers gRPC-Web plugin
```

Comment:
these test cases don't show the trailers, can you add a test case that shows them?